diamm/helpers/solr_pagination.py

```diff
@@ -148,6 +144,16 @@ def type_list(self):
         return OrderedDict(filtered_facets)
 
     @property
+    def images_count(self):
```

Comment:
Perhaps this could be inserted into the `types` list with a key (`sources_with_images`) instead of `image_count` being at the root level.